### PR TITLE
rcqt module: Refactoring

### PR DESCRIPTION
**Pull request by jkottiku**
1. Move validatePackages() to PackageHandler class.
2. Code Indent.

### CODE REVIEW

File: rcqt.so/include/debPackageInfo.h [hunk 1]
```diff
@@ -6,9 +6,13 @@ class DebPackageInfo : public PackageInfo{
 public:
 	DebPackageInfo(const std::string& pkgname,
                  const std::string& pkgmgr,
-								 const std::string& cmd):PackageInfo(pkgname,
+								 const std::string& cmd1,
+								 const std::string& cmd2,
```
Comment (manoj-freyr) [inline rcqt.so/include/debPackageInfo.h:10]: having a vector<string> is better here?

File: rcqt.so/src/packageHandler.cpp [hunk 2]
```diff
@@ -25,3 +25,44 @@ bool PackageHandler::parseManifest(){
 	//std::cout << m_pkgversionmap.size() << " packages to validate " << std::endl;
 	return true;
 }
+
+void PackageHandler::validatePackages(){
+  std::cout << "MANOJ: file nameis " << m_manifest << std::endl;
```
Comment (manoj-freyr) [inline rcqt.so/src/packageHandler.cpp:30]: this we can remove the reference to name

**Review by manoj-freyr — APPROVED** [review]
LGTM

**Review by manoj-freyr — APPROVED** [review]
Looks good.

File: rcqt.so/include/debPackageInfo.h (hunk 1 above)
Comment (jkottiku) [reply rcqt.so/include/debPackageInfo.h]: Yes, agree. Done.

**Pull request merged**